packages/react-app/src/services/bounty.service.ts

```diff
@@ -15,8 +15,8 @@ export const getFilters = (query: NextApiQuery): FilterParams => {
 	 */
 	const filters = {} as FilterParams;
 
-	if (typeof query.status === 'string') filters.status = query.status;
-	if (typeof query.paidStatus === 'string') filters.paidStatus = query.paidStatus;
+	if (typeof query.status === 'string') filters.status = query.status ? query.status.split(',') : [];
+	if (typeof query.paidStatus === 'string') filters.paidStatus = query.paidStatus ? query.paidStatus.split(',') : [];
```

Comment:
What are the expected inputs into `query.paidStatus`?

Reply:
Possible inputs:
1. []
2. [Paid]
3. [Unpaid]
4. [Paid, Unpaid]

As you correctly pointed out in the comment below, the first and the 4th one are doing the same job